### DISCUSSION TITLE
Is databend suitable for point query?

### DISCUSSION THREAD

**Discussion by tempbottle** [databendlabs/databend#13964] [General]
Scenario like this:
[Doris point query](https://doris.apache.org/zh-CN/docs/query-acceleration/hight-concurrent-point-query/)

**Comment by bohutang** [comment]
Databend default build bloomfilter index for each column, so it's very efficient for point query.

**Reply by tempbottle** [reply]
Any benchmark data for this?

**Reply by bohutang** [reply]
No, it's fast enough we build blooming filter for each block if the point query route to a certain block.

**Reply by tempbottle** [reply]
Great! I'll give it a try.
